providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/grpc/GrpcConnector.java

```diff
@@ -37,7 +37,7 @@ public class GrpcConnector {
     private final long deadline;
 
     private final Cache cache;
-    private final Consumer<ProviderState> stateConsumer;
+    private final BiConsumer<ProviderState, List<String>> stateConsumer;
```

Comment:
For now, this is fine, but we should consider whether it makes sense to just use a Consumer and a POJO to provide more information. Changing the amount of parameters includes a lot of changes. A proper POJO with a good builder could also reduce this heavily in the future.